src/main/resources/application.properties

```
@@ -1 +1,5 @@
-
+#DATABASE CONFIG
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/wedding_halls
+spring.datasource.username=user
+spring.datasource.password=user
```

Look how convenient is to have environment variable for user, password. You change it once, and it's changed both for docker, and for spring:
```suggestion
spring.datasource.username=${WEDDING_HALL_DB_USER}
spring.datasource.password=${WEDDING_HALL_DB_PASSWORD}
```